### DISCUSSION TITLE
Changing carts question

### DISCUSSION THREAD

**Discussion by JoeBuk724** [sanni/cartreader#968] [General Chat]
When I've been changing carts I always turn the device off, regardless if the new cart is for a different system or the same system. I'm curious, is that necessary? Is it safe to just hit reset and swap cartridges while in the main menu? If so, is that also safe when having to change the 3v/5v switch? Thanks!

**Comment by cmcm711** [comment]
It's safe to unplug after resetting, but safer to turn it off.
